### PR TITLE
logind: add BlockWeakInhibited property

### PR DESCRIPTION
**Pull request by bluca**
Fixes https://github.com/systemd/systemd/issues/34091
Follow-up for 804874d26ac73e0af07c4c5d7165c95372f03f6d

### CODE REVIEW

File: man/org.freedesktop.login1.xml [hunk 1]
```diff
@@ -805,9 +808,10 @@ node /org/freedesktop/login1 {
       timestamps of the last change of the idle hint boolean, in <constant>CLOCK_REALTIME</constant> and
       <constant>CLOCK_MONOTONIC</constant> timestamps, respectively, in microseconds since the epoch.</para>
 
-      <para>The <varname>BlockInhibited</varname> and <varname>DelayInhibited</varname> properties encode
-      the currently active locks of the respective modes. They are colon separated lists of
-      <literal>shutdown</literal>, <literal>sleep</literal>, and <literal>idle</literal> (see above).</para>
+      <para>The <varname>BlockInhibited</varname>, <varname>BlockWeakInhibited</varname> and
```
Comment (keszybz) [inline man/org.freedesktop.login1.xml]: Oxford comma please, esp. that that's used in the next sentence.

**Review by keszybz — APPROVED** [review]

**Comment by poettering** [comment]
i wished you would stop doing that "fixes:" from prs to issues where only a small subset of issues there is fixed. use something else, such as "See:".

because again that issue will now be closed when this pr is merged, and noone will notice that the most of the listed issues remain unaddressed.

**Comment by bluca** [comment]
> i wished you would stop doing that "fixes:" from prs to issues where only a small subset of issues there is fixed. use something else, such as "See:".

Github wouldn't recognize that and thus won't link the PR and the issue, which makes my life exponentially harder when tracking the status of the milestones. Clicking the 'reopen' button on the other hand takes less than a second.

**Comment by poettering** [comment]
> Github wouldn't recognize that and thus won't link the PR and the issue, which makes my life exponentially harder when tracking the status of the milestones. Clicking the 'reopen' button on the other hand takes less than a second.

Of course it links them. If you mention some other PR/issue in a PR/issue it will generate an entry on the other one. That should be absolutely enough.

**Pull request merged**